src/content/about.md

```diff
@@ -0,0 +1,239 @@
+task runner), [Gulp](https://gulpjs.com/) (streams and pipes) or [Webpack](https://webpack.js.org/) (a module 
+bundler), these all aim to solve different problems than what Broccoli what built for.
+
+Broccoli provides a modular plugin API to leverage other Node tools. Tools like [Babel](https://babeljs.io/),
+[Rollup](https://rollupjs.org/), [Node-Sass](https://github.com/sass/node-sass), 
+[Webpack](https://webpack.js.org/), [Browserify](http://browserify.org/), and many more can easily be used with 
+Broccoli. These tools perform the actual work by transforming files, Broccoli merely connects inputs to outputs.
+
+## Thinking in Broccoli
+
+There are 3 main concepts to get your head around when using Broccoli:
+
+* [Directories](#directories)
+* [Plugins](#plugins)
+* [Trees](#trees)
+
+### Directories
+
+Using the above tools in a standalone fashion is relatively straight forward, they read input files, and write 
+to output files. The difficulty comes when connecting different tools together as you need to manage all the 
+interim state yourself. The only common interface between them is the file system.
+
+Say we want to concatenate our JavaScript files, minify them, and copy the results to our build directory:
+
+```js
+const fs = require('fs');
+const path = require('path');
+const minify = require('minify');
+const readDir = require('./readDir');
+const readFiles = require('./readFiles');
+
+function concatFiles(sourceDir, outputFile) {
+    const output = readDir(sourceFiles).reduce((output, file) => output += `;${file.content}`, '');
+    fs.writeFileSync(outputFile, output)
+}
+
+function minifyFiles(sourceFiles, outputDir) {
+    return readFiles(sourceFiles).map(file => {
+        const minified = minify(file.path);
+        const outputFile = `${outputDir}/${file.name}.min.${file.extension}`;
+        fs.writeFileSync(outputFile, minified)
+        return outputFile;
+    })
+}
+
+function copyFiles(sourceFiles, outputDir) {
+    sourceFiles.forEach(filePath => {
+        const file = path.basename(filePath);
+        fs.createReadStream(path).pipe(fs.createWriteStream(`${outputDir}/${file}`));
+    });
+}
+
+const concatted = concatFiles('app', '/tmp/concat-files')
+const minified = minifyFiles(concatted, '/tmp/minified-files')
+copyFiles(minified, './build');
+```
+
+The above is all well and good, but notice how we have to handle the state between each operation. Now imagine how
+this scales as our build pipeline grows, adding in Sass compilation, JavaScript transpilation (Babel), 
+fingerprinting, etc. That's lots of temp directories and file state that you have to handle.
+
+Broccoli works by managing of a set of directories, connected together by plugins, which describe how files 
+are moved or transformed at each step of the build process. Broccoli ensures plugins are called in the prescribed
+order, and writes the result to a `target` directory. Each plugin is responsible for processing files passed to 
```

Comment:
It might be to low level, but an interesting fact, is if you use the same plugin instance twice, it doesn't result in duplicate work, as plugins auto memoize on a per rebuild basis.

feel free to disregard if this is the wrong place to mention this.

Reply:
I feel like that's too low level